[locator.py]
  [test_find_by_locator[innerText-foobarBARbaz\]]
    bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1869538
    expected: FAIL

  [test_find_by_inner_text[ignore_case_true_full_match_no_max_depth\]]
    bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1869538
    expected: FAIL

  [test_find_by_inner_text[ignore_case_false_full_match_no_max_depth\]]
    bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1869538
    expected: FAIL

  [test_find_by_inner_text[ignore_case_true_partial_match_no_max_depth\]]
    bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1869538
    expected: FAIL

  [test_find_by_inner_text[ignore_case_false_partial_match_no_max_depth\]]
    bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1869538
    expected: FAIL

  [test_find_by_inner_text[ignore_case_true_full_match_max_depth_zero\]]
    bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1869538
    expected: FAIL

  [test_find_by_inner_text[ignore_case_false_full_match_max_depth_zero\]]
    bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1869538
    expected: FAIL

  [test_find_by_inner_text[ignore_case_true_partial_match_max_depth_zero\]]
    bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1869538
    expected: FAIL

  [test_find_by_inner_text[ignore_case_false_partial_match_max_depth_zero\]]
    bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1869538
    expected: FAIL

  [test_find_by_inner_text[ignore_case_true_full_match_max_depth_two\]]
    expected: FAIL

  [test_find_by_inner_text[ignore_case_false_full_match_max_depth_two\]]
    expected: FAIL

  [test_find_by_inner_text[ignore_case_true_partial_match_max_depth_two\]]
    expected: FAIL

  [test_find_by_inner_text[ignore_case_false_partial_match_max_depth_two\]]
    expected: FAIL

  [test_locate_by_context[same_origin\]]
    expected: FAIL

  [test_locate_by_context[cross_origin\]]
    expected: FAIL

  [test_locate_by_context_in_iframe[same_origin\]]
    expected: FAIL

  [test_locate_by_context_in_iframe[cross_origin\]]
    expected: FAIL

  [test_locate_by_context_in_shadow_dom[open-same_origin\]]
    expected: FAIL

  [test_locate_by_context_in_shadow_dom[open-cross_origin\]]
    expected: FAIL

  [test_locate_by_context_in_shadow_dom[closed-same_origin\]]
    expected: FAIL

  [test_locate_by_context_in_shadow_dom[closed-cross_origin\]]
    expected: FAIL
